### PR TITLE
Add cinder registry volume creation to Openstack install

### PR DESCRIPTION
**Pull request by day4skiing**
#### What does this PR do?
Brief explanation of the code or documentation change you've made

#### How should this be manually tested?
Include commands to run your new feature, and also post-run commands to validate that it worked. (please use code blocks to format code samples)

add the following block to `all.yml`
and run `end-to-end.yml` playbook
```
openshift_hosted_registry_storage_kind: openstack
openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
openshift_hosted_registry_storage_openstack_filesystem: ext4
openshift_hosted_registry_storage_volume_size: 50Gi

osp_volumes:
- name: "registry-vol"
  display_description: "Registry Volume"
  display_name: "registry-vol"
  size: 50
  purpose: "ose_registry"
```

#### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
No
#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

**Comment by day4skiing** [comment]
PTAL

File: inventory/sample.osp.example.com.d/inventory/group_vars/all.yml [hunk 1]
```diff
@@ -90,3 +90,17 @@ rhsm_password: ''
 #rhsm_password: ''
 # leave commented out if you want to `--auto-attach` a pool
 #rhsm_pool: ''
+
+# NOTE: these need to be declared in this file because openshift_hosted_registry_storage_kind
+# is needed prior to the creation of OSEv3 hosts. The remainder are here for grouping purposes
+openshift_hosted_registry_storage_kind: openstack
+openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
+openshift_hosted_registry_storage_openstack_filesystem: ext4
+openshift_hosted_registry_storage_volume_size: 50Gi
```
Comment (oybed) [inline inventory/sample.osp.example.com.d/inventory/group_vars/all.yml]: The `volume_size` should be set based on the `osp_volumes` values (and hence removed from here)

File: playbooks/openshift/cinder-registry.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Create a volume for the registry
+- hosts: localhost
+  roles: 
+  - infra-ansible/roles/osp/admin-volume
+
+- hosts: OSEv3
+  tasks:
+  - name: "set registry volumeID fact"
+    set_fact: 
+      openshift_hosted_registry_storage_openstack_volumeID: "{{ disk.volume.id }}"
```
Comment (oybed) [inline playbooks/openshift/cinder-registry.yml]: Per comment above - just add `openshift_hosted_registry_storage_volume_size: "{{ disk.size }}Gi"` here

Comment (oybed) [reply playbooks/openshift/cinder-registry.yml]: ... there may be a better fact to use than `disk.volume.id`, but you probably have the capture so you can decide which one it is.

Comment (day4skiing) [reply playbooks/openshift/cinder-registry.yml]: Yup, `disk.volume.size`

Comment (oybed) [reply playbooks/openshift/cinder-registry.yml]: Thx. Code change looks good. Still need to update the doc and example inventory. Could be good to call-out that these two facts will be automatically set.

Comment (day4skiing) [reply playbooks/openshift/cinder-registry.yml]: Docs updated -- PTAL

**Pull request merged**